glamorous/button.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import glamorous, { Div } from 'glamorous';
```

Comment:
This is unfortunately incorrect (and I don't think it should even work). Because `glamorous` does not _actually_ export _anything_ but the `glamorous` function, which happens to have properties for all the components and component factories. So you shouldn't be able to do this. Instead, this should be:

```javascript
import glamorous from 'glamorous'

const {Div} = glamorous
```